Makefile

```diff
@@ -1,19 +1,23 @@
-ddown = docker-compose down
-
-install: build migrate
+# General =====================================================================
+install: build migrate import_initial_data
 
 build:
 	docker-compose build
 
 up:
 	docker-compose up
 
+ddown = docker-compose down --remove-orphans
 down:
-	$(ddown) --remove-orphans
+	$(ddown)
 
 manage:
 	docker-compose run --rm web python manage.py $(command)
 
+import_initial_data:
+	docker-compose run --rm web python manage.py import_initial_data ./mars_api/data_import/terra-mars-initial-data.csv
```

Comment:
You can probs add an alias for `docker-compose run --rm web python manage.py` or sinnce it's used in a couple of places.